### PR TITLE
Release

### PR DESCRIPTION
**Pull request by adebsa2401**
In this PR,

- I created a splash screen
- I created a signup page
- I created a login page
- I created categories index page
- I created categories form creation form page
- I created transactions index page
- I created transactions creation form page
- I implement unit and integrations tests

Demo video link: [here](https://www.loom.com/share/cb73a9cc0dbb4eb0af5442db6d1b50c5)
Live link: [here](https://budget-app-4bt4.onrender.com/)

Cheers

### CODE REVIEW

File: spec/models/category_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  it 'should fail if name is too short' do
+    @category.name = 'Fo'
+    expect(@category).to_not be_valid
+  end
+
+  it 'should fail if total_amount is not equal to the sum of operations' do
+    login_as @user
+    expect(@category.total_amount).to eq(@operation1.amount + @operation2.amount)
+  end
```
Comment (Deepakdanger) [inline spec/models/category_spec.rb]: - [  ] One test is failing. Please fix that

Screenshot from 2023-03-22 23-56-56

File: README.md [hunk 2]
```diff
@@ -1,24 +1,268 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Deepakdanger) [inline README.md]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to Gregoire Vella on Behance the author of the original design, as required by the Creative Commons license of the design.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @adebsa2401, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required changes
_Check the comments under the review._


## Optional changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by andersonlebon — APPROVED** [review]
## Hi @adebsa2401 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**